adafruit_esp32spi/adafruit_esp32spi.py

```diff
@@ -768,3 +769,12 @@ def get_time(self):
             resp = self._send_command_get_response(_GET_TIME)
             return struct.unpack('<i', resp[0])
         raise RuntimeError("Must be connected to WiFi before obtaining NTP.")
+
+    def get_remote_data(self, socket_num):
+        """
+        Method to obtain remote IP
+        """
+        self._socknum_ll[0][0] = socket_num
+        resp = self._send_command_get_response(_GET_REMOTE_DATA_CMD,
+                                               self._socknum_ll, reply_params=2)
+        return resp[0]
```

Comment:
the `get_remote_data` command in NINA returns 2 reply params. The first is the remote IP, the second is the remote port

instead of just returning the IP, I would return both. 

something like

`return { 'ip_addr': resp[0], 'port': resp[1] }`

Reply:
File "/lib/adafruit_esp32spi/adafruit_esp32spi.py", line 550, in pretty_ip
KeyError: 0

Reply:
So by changing the return type an dictionary object means that when you call this method you will need to do an additional step to get the ip whenever you call this method. Dictionary objects have a method on it called `.get(keyname)`. You'll need to update usages of this method so you actually grab the IP address from the dictionary.

Example:
```
remote_data = esp.get_remote_data(socknum)
Ip = remote_data.get('ip_addr')
```